test/OpenSTValue.js

```diff
@@ -472,4 +473,34 @@ contract('OpenSTValue', function(accounts) {
 			})
 		})
 	})
+
+	describe('ProcessStaking with fallback', async () => {
```

Comment:
To be consistent, the "test stories" at the top should be updated to reflect this addition.

Reply:
updated in OpenSTValue.js and OpenSTUtility.js